pkgs/tools/system/bottom/default.nix

```diff
@@ -19,7 +19,7 @@ rustPlatform.buildRustPackage rec {
 
   buildInputs = lib.optional stdenv.hostPlatform.isDarwin darwin.apple_sdk.frameworks.IOKit;
```

Comment:
darwin should not be used as an input. Instead IOKit should be inputed and inherited in top-level.